css/drawing-tool.styl

```diff
@@ -20,6 +20,9 @@
   .clickable
     cursor: pointer
 
+  .clickable-line
```

Comment:
In Stylus, I think you can combine this with the preceding declaration by indenting and using `&-line`.